client/themes/crypto.css

```diff
@@ -9,75 +9,79 @@ GitHub: https://github.com/aynik
 */
 
 @font-face {
-    font-family: Inconsolata-g;
-    src: url("fonts/inconsolatag.woff") format("woff"), url("fonts/inconsolatag.ttf") format("ttf");
+	font-family: Inconsolata-g;
```

Comment:
Uuuurgh, no, please no... :-)

`client/css/style.css` is 2-space indentation, let's use that instead of tabs please. Plus it will reduce the change list a fair bit!